03.asynchronous/callback.js

```diff
@@ -0,0 +1,36 @@
+import sqlite3 from "sqlite3";
+import timers from "timers/promises";
+
+const db = new sqlite3.Database(":memory");
+
+const executeWithoutError = () =>
+  db.run(
+    "CREATE TABLE books(id INTEGER PRIMARY KEY AUTOINCREMENT, title TEXT NOT NULL UNIQUE )",
+    () =>
+      db.run("INSERT INTO books(title) values ('蟹工船')", function () {
+        console.log(`id: ${this.lastID}`);
+        db.get("SELECT * FROM books WHERE title = '蟹工船' ", (error, row) => {
+          console.log(`id: ${row.id} title: ${row.title}`);
+          db.run("DROP TABLE books");
+        });
+      }),
+  );
+
+const executeWithError = () =>
+  db.run(
+    "CREATE TABLE books(id INTEGER PRIMARY KEY AUTOINCREMENT, title TEXT NOT NULL UNIQUE )",
+    () =>
+      db.run("INSERT INTO books(title) values (null)", (error) => {
```

Comment:
SQL のキーワードは大文字で記述してください。

```suggestion
      db.run("INSERT INTO books(title) values (NULL)", (error) => {
```

Reply:
SQLIteのキーワードを大文字で表記するよう修正しました！
また、`VALUES`もキーワードになっていましたので合わせて大文字に修正しました！

[SQLite Keywords](https://www.sqlite.org/lang_keywords.html)

[SQLiteのキーワードを大文字で表記するよう修正](https://github.com/tyrrell-IH/js-practices/pull/3/commits/0bc80392cf6c49df0bcb0eee11751acf77e77e9f)